### DISCUSSION TITLE
Very low volume

### DISCUSSION THREAD

**Discussion by GameTimeWarp** [frntc/Sidekick64#71] [General]
So I'm getting extremely low volume on SID/FM emulation. I have everything cranked up to 15 in the volume settings on the Sidekick64 and have tried both HDMI out as well as the 3.5mm jack out. I have to crank up my system speakers to MAX and even then the audible volume I'm getting out is on the low side for sure. Ideas?

**Comment by frntc** [comment]
Maybe the panning settings cause some rescaling, you could try at set everything to center for testing. Apart from that, there should be nothing you can do wrong :-) The audio jack is known to be of low volume, but HDMI should work (maybe try another HDMI device?)

**Reply by GameTimeWarp** [reply]
I tried all combination of settings in the audio settings in the Sidekick and still get very low audio. I noticed when changing the volume settings that the only value that really seems to make any difference at all is if I set them to 0 which then turns the audio off for that specific device. Anywhere from 1-15 doesn't seem to change anything as far as volume goes. There is also some sort of low audio distortion going on like something is set too loud. When checking out the LCD on the Sidekick and clicking the left button to change the view to the 3 channel power meter view, the top 2 always stay well within range but the bottom most meter is almost always pegged in the red. I'm not sure if these 2 issues are related but it sure feels like it. HDMI out to another device produced the same results. I sure feels like it might be something on the software side. Let me know how else I may be able to help with reproducing this on your side.

**Comment by blindguy** [comment]
I'm having almost the same issue.  When I use the 3.5 mm jack to some external speakers i have to turn it up all the way to get the 8 sids to work and it is still VERY low volume.  I can do HDMI out to play mods and it is fine but HDMI out to play the 8 SID does not get any volume.  Any suggestions are appreciated!

**Comment by StmLord** [comment]
I can confirm this, PWM output on 8sids setup is whispering silence, I tried with a cheap PC active speakers and at the full volume I could just hear it whispering the tune, not low, but whispering! 
Btw, I have used the same Pi with that output on raspberry Pi OS and the volume is almost as high as the output of my Mac.

**Comment by frntc** [comment]
You're right, it seems that something in the 8-SID HDMI output is broken (after used that even <8 SIDs via HDMI are silenced). Need to have a look. For 8-SIDs you can use https://github.com/frntc/Sidekick64/releases/tag/v0.5 meanwhile.

PWM volume is low (but it plays ok-ish here, no whispering) -- does the oscilloscope/peak meter on the TFT show full levels? If yes, making the output louder would cause output values to be clamped.

**Reply by GameTimeWarp** [reply]
v0.5 fixes HDMI volume output for sure on my end. Nothing more can be tried to get PMW output fixed so it outputs louder? I would love to get PMW fixed because I can run the audio out direct into my Retrotink 5x which then lets me see what's going on (browse and select new files) because I have C64 video on the screen. It's a pain switching between inputs everytime I wanna browse for and start a new song.
